[Bug]: CoreWCF.BuildTools.Roslyn generates CS9057 compiler warning when using .NET SDK 8

### Duplicate ?

- [x] I have searched issues/discussions and did not find other issues/discussions reporting this bug.

### Product version

1.7.0

### Describe expected behavior

CoreWCF v1.7.0 should not create compiler warnings when building with .NET SDK 8.

### Describe actual behavior

I am using .NET SDK 8.0.405 and .NET 8 as target framework.

After updating CoreWCF to the latest release 1.7.0 i've encountered the following build warning:

CSC : warning CS9057: The analyzer assembly 'XXX\.nuget\packages\corewcf.primitives\1.7.0\analyzers\dotnet\roslyn4.0\cs\CoreWCF.BuildTools.Roslyn4.0.dll' references version '4.
12.0.0' of the compiler, which is newer than the currently running version '4.11.0.0'. [XXX\CoreWcf.SDK8\CoreWcf.SDK8.csproj]

Not sure if you would consider this as a bug.

### Which binding

BasicHttp

### security

None

### Which .NET version

.NET 8

### Which os platform

Windows

### Code snippet used to reproduce the issue

```c#
csproj

<Project Sdk="Microsoft.NET.Sdk">

    <PropertyGroup>
        <OutputType>Exe</OutputType>
        <TargetFramework>net8.0</TargetFramework>
        <ImplicitUsings>enable</ImplicitUsings>
        <Nullable>enable</Nullable>
    </PropertyGroup>

    <ItemGroup>
      <PackageReference Include="CoreWCF.Http" Version="1.7.0" />
      <PackageReference Include="CoreWCF.Primitives" Version="1.7.0" />
    </ItemGroup>

</Project>

global.json

{
  "sdk": {
    "version": "8.0.0",
    "rollForward": "latestMinor",
    "allowPrerelease": false
  }
}
```

### Stacktrace if any

```shell

```

--- a/src/CoreWCF.BuildTools/src/CoreWCF.BuildTools.Roslyn4.0.csproj
+++ b/src/CoreWCF.BuildTools/src/CoreWCF.BuildTools.Roslyn4.0.csproj
@@ -10,7 +10,7 @@
     <Compile Remove="*.Roslyn3.11.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" VersionOverride="4.12.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" VersionOverride="4.8.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" PrivateAssets="all" />
   </ItemGroup>
 </Project>

